templates/site/ide.html

```diff
@@ -4,7 +4,7 @@
 {% block content %}
     {% include "partials/banner.html" %}
     {% with %}
-        {% set img_src = "images/logos/jupyterhublogo.png" %}
```

Comment:
Please delete this file if it is no longer used.

Reply:
Image deleted